arkouda/strings.py

```diff
@@ -562,8 +590,9 @@ def strip(self, chars: Optional[Union[bytes, str_scalars]] = "") -> Strings:
         """
         if isinstance(chars, bytes):
             chars = chars.decode()
-        args = {"objType": self.objtype, "name": self.entry.name, "chars": chars}
-        rep_msg = generic_msg(cmd="segmentedStrip", args=args)
+        rep_msg = generic_msg(
+            cmd="segmentedStrip", args={"objType": self.objtype, "name": self.entry.name, "chars": chars}
```

Comment:
the `.name` on `self.entry.name` in not needed right?

Reply:
Correct. I will remove that.

Reply:
Removed.